spec/examples/util/i18n.spec.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env mocha */
```

Comment:
@schraj I hate to be a Major Payne here, but I’m actually in the process of transitioning everything from Mocha to Tape, and generally I’d like to avoid creating new Mocha specs. Would you mind porting this over to a Tape unit test in the `test/` directory?

![](https://i.ytimg.com/vi/KclWPUBC9sM/movieposter.jpg)